src/assets/i18n/fr_FR.json

```diff
@@ -756,7 +756,7 @@
   "modal.heading.remove.order.template": "Supprimer le produit du modèle de commande",
   "order.tracking.error": "Malheureusement, nous n’avons pas pu localiser de commande avec les informations que vous nous avez fournies.",
   "order_template.create.heading": "Créer un modèle de commande",
-  "payment.error.PaymentInstrumentAlreadyExists": "L’instrument de paiement n’a pas pu être créé. Les données de paiement avec les paramètres fournis existent déjà.",
+  "payment.error.PaymentInstrumentAlreadyExists": "Le moyen de paiement n’a pas pu être mis en place. Les données de paiement avec les paramètres fournis existent déjà.",
```

Comment:
If you change the translation for (payment instrument - Zahlungsmittel) here - it should be done for all other occurrances, too. Pls note the difference between payment method and payment instrument.

Reply:
You are right. We will change "l'instrument de paiement" to "le moyen de paiement" at all instances occurring in the localization file within beginning of this week. Please wait untilthis is done before merging.

Reply:
Done